libraries/SITL/SIM_Vicon.cpp

```diff
@@ -104,6 +107,22 @@ void Vicon::update_vicon_position_estimate(const Location &loc,
     float yaw;
     attitude.to_euler(roll, pitch, yaw);
 
+#if USE_VISION_POSITION_ESTIMATE
+    // use the more recent VISION_POSITION_ESTIMATE message
+    mavlink_msg_vicon_position_estimate_pack_chan(
```

Comment:
that is exactly the same as already send, right ?

Reply:
yeah ... fixed :-)